docs/source/installation.md

```diff
@@ -11,7 +11,5 @@ $ wget https://github.com/livepeer/go-livepeer/releases/download/<RELEASE_VERSIO
 After downloading the file, untar the archive and move the `livepeer` binary so that it is executable within your `$PATH`:
 
 ```
-$ tar -zxvf livepeer-<YOUR_PLATFORM>-amd64.tar.gz
-$ mv livepeer-<YOUR_PLATFORM>-amd64/livepeer /usr/local/bin
-$ mv livepeer-<YOUR_PLATFORM>-amd64/livepeer_cli /usr/local/bin
+$ tar -C /usr/local/bin -fxz livepeer-<YOUR_PLATFORM>-amd64.tar.gz
```

Comment:
This doesn't work for me on macOS. Likely because the result of untarring the archive is not the binaries, but a folder containing the binaries.

Reply:
This was from Livepool where I don't pack it into a folder. 

Fixed this in the latest commit by stripping the folder names. 

Would like to point out however that MacOS users can't use these instructions anyway as wget isn't installed on MacOS by default, so if these docs were to be used for MacOS then we might want to switch that to `curl -o`